packages/common/src/doctype.ts

```diff
@@ -18,56 +18,85 @@ export enum AnchorStatus {
     NOT_REQUESTED, PENDING, PROCESSING, ANCHORED, FAILED
 }
 
-/**
- * Describes anchor record
- */
-export interface AnchorRecord {
-    prev: CID; // should be CID type
-    proof: CID; // should be CID type
-    path: string;
+export interface RecordHeader {
+    controllers: Array<string>
+    schema?: string
+    tags?: Array<string>
+
+    unique?: string
+
+    [index: string]: any // allow support for future changes
```

Comment:
Shouldn't future changes update this interface anyways?

Reply:
makes sense! I'll update it

Reply:
I guess we have to leave this for now because of the `nonce` in `TileDoctype`.

Reply:
yeah, this should be flexible